css/styles.css

```
@@ -0,0 +1,2205 @@
+  --acrylic-bg-light: linear-gradient(to right bottom,
+      rgba(var(--color-primary-rgb), 0.05),
+      rgba(var(--color-primary-rgb), 0.01));
+  --acrylic-bg-dark: linear-gradient(to right bottom,
+      rgba(var(--color-primary-rgb), 0.1),
+      transparent 70%);
+  --acrylic-bg: var(--acrylic-bg-dark);
+  --acrylic-blur: saturate(180%) blur(20px);
+  --neon-shadow: 0 0 8px rgba(var(--color-primary-rgb), 0.7);
+  --color-shadow-light: 0 1.6px 3.6px 0 rgba(0, 0, 0, 0.132), 0 0.3px 0.9px 0 rgba(0, 0, 0, 0.108);
+  --color-shadow-dark: 0 1.6px 3.6px 0 rgba(0, 0, 0, 0.22), 0 0.3px 0.9px 0 rgba(0, 0, 0, 0.18);
+  --color-shadow: var(--color-shadow-dark);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--color-shadow is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)